### PR TITLE
fix:Added sponsor placeholder

### LINKED ISSUE #1775: Addition Of Placeholder in Sponsors
**Actual Behaviour**
Doesn't Exist now.

**Expected Behaviour**
Should have a placeholder.

**Would you like to work on the issue?**
Yes

**Target Milestone to solve the issue**

https://github.com/fossasia/open-event-android/milestone/31

### PR DESCRIPTION
**Pull request by shreyasnbhat**
Fixes #1775 
@arp95 @iamareebjamal @Shailesh351 Please review.
Give me a solution to the resizing if possible. Tried out what @arp95 said. The image from picasso just overlays on top of the distorted background.

### CODE REVIEW

**Comment by the-dagger** [comment]
@shreyasnbhat Could you remove unused imports?
I've turned off unnecessary errors reported by codacy so now it only reports actual issues with the code.

Try and keep the codacy error to a minimum.

**Comment by shreyasnbhat** [comment]
Updated the PR. Please review now.

File: android/app/src/main/java/org/fossasia/openevent/adapters/SponsorsListAdapter.java [hunk 1]
```diff
@@ -100,14 +100,14 @@ public void onBindViewHolder(final RecyclerView.ViewHolder holder, int position)
...
-                sponsorViewHolder.sponsorImage.setVisibility(View.VISIBLE);
                 OpenEventApp.picassoWithCache
                         .load(Uri.parse(logo))
                         .resize(width, (height / 6))
+                        .placeholder(R.drawable.ic_sponsors_black_24dp)
                         .centerInside()
                         .into(sponsorViewHolder.sponsorImage);
             } else {
-                sponsorViewHolder.sponsorImage.setVisibility(View.GONE);
+                sponsorViewHolder.sponsorImage.setImageDrawable(context.getResources().getDrawable(R.drawable.ic_sponsors_black_24dp));
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/adapters/SponsorsListAdapter.java]: Shouldn't show an image if no image is associated with the target

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/org/fossasia/openevent/adapters/SponsorsListAdapter.java (hunk 1 above)
Comment (shreyasnbhat) [reply android/app/src/main/java/org/fossasia/openevent/adapters/SponsorsListAdapter.java]: Alright. Will change this.

File: android/app/src/main/java/org/fossasia/openevent/adapters/SponsorsListAdapter.java [hunk 2]
```diff
@@ -100,10 +100,10 @@ public void onBindViewHolder(final RecyclerView.ViewHolder holder, int position)
             sponsorViewHolder.sponsorType.setText(sponserType);
             sponsorViewHolder.sponsorName.setText(sponserName);
             if(logo != null) {
-                sponsorViewHolder.sponsorImage.setVisibility(View.VISIBLE);
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/adapters/SponsorsListAdapter.java]: If this is removed, the next viewholder will not show the image even if the URL is available

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

**Pull request merged**